src/main/java/com/palantir/gradle/graal/GradleGraalPlugin.java

```diff
@@ -76,6 +76,17 @@ public final void apply(Project project) {
                     task.dependsOn(downloadGraal);
                 });
 
+        TaskProvider<ReflectionConfigTask> reflectionConfig = project.getTasks().register(
```

Comment:
it'd be more appropriate to split this task into a separate PR from the options addition

Reply:
(more broadly, I'd prefer a different strategy to humans adding files to a list, happy to talk about the specifics offline)

Reply:
OK, happy to split that into a separate PR.  Agreed that manually curating that list is not sustainable. The strategy I'm leaning towards to generate this reflectconfig class list is to run a little JavaExec task which uses some reflection to find all classes annotated with @JsonProperty, @JsonSerialize etc.

Reply:
I think we'd be better off with an explicit "GraalReflectable" annotation + annotation processor than specifically scanning for Jackson annotations.

Reply:
...and that shouldn't be part of this repo, or generative of a Java class, but instead something that generates the Graal-format config file.